irc/config.go

```diff
@@ -229,7 +229,8 @@ type Config struct {
 	}
 
 	Datastore struct {
-		Path string
+		Path        string
+		AutoUpgrade *bool
```

Comment:
Just curious, why is this `*bool` instead of just `bool`? If the argument is around defaults, I'd expect the config on upgrading to behave as it did before (that is, no automagic upgrade). If they read the upgrade notes, they'll see the new `auto-upgrade` key and be able to set it.

To ensure they know, I'll call explicit attention to it in the upgrade message.